docs/cover.out

```diff
@@ -4,67 +4,15 @@ github.com/vatsal278/msgbroker/pkg/parser/parser.go:18.2,19.16 2 1
 github.com/vatsal278/msgbroker/pkg/parser/parser.go:22.2,22.12 1 1
 github.com/vatsal278/msgbroker/pkg/parser/parser.go:14.16,16.3 1 0
 github.com/vatsal278/msgbroker/pkg/parser/parser.go:19.16,21.3 1 1
+github.com/vatsal278/msgbroker/pkg/validate/validate.go:5.36,8.17 3 1
+github.com/vatsal278/msgbroker/pkg/validate/validate.go:11.2,11.12 1 1
+github.com/vatsal278/msgbroker/pkg/validate/validate.go:8.17,10.3 1 1
 github.com/vatsal278/msgbroker/internal/pkg/parser/parseRequest.go:10.68,12.16 2 1
 github.com/vatsal278/msgbroker/internal/pkg/parser/parseRequest.go:15.2,16.16 2 1
 github.com/vatsal278/msgbroker/internal/pkg/parser/parseRequest.go:19.2,19.12 1 1
 github.com/vatsal278/msgbroker/internal/pkg/parser/parseRequest.go:12.16,14.3 1 1
 github.com/vatsal278/msgbroker/internal/pkg/parser/parseRequest.go:16.16,18.3 1 1
 github.com/vatsal278/msgbroker/pkg/responseWriter/responseWriter.go:14.104,22.2 5 1
-github.com/vatsal278/msgbroker/internal/router/router.go:11.27,22.2 8 1
```

Comment:
Removal of these would suggest that the files would not be visible in the coverage report.
Check using the covertool that all files are shown.

Reply:
As expected, coverage for removed lines for the corresponding files are not shown. 
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/77770177/208080611-711d5c84-3447-4c45-86e2-d62b5a9be575.png">
Regenerate the coverage report and this should also adhere to the README steps mentioned to view the coverage.